tests/examples/externals/definitions.py

```diff
@@ -0,0 +1,47 @@
+from dusk.script import *
+
+# A `definitions.py` could be one way to naturally introduce symbols that can
+# be imported and used in dusk stencils.
+
+regions = Regions(
```

Comment:
You call them regions ... but aren't them actually markers of where regions start?
One could be tricked to think that by just providing `lb` the DSL let's you compute on the later boundary, whereas you need to specify start = `lb` and end = `nudging`.

Reply:
true. I took this mainly from the original proposal which had:
```python
class region(Enum):
    lb = 0, 
    nudging = 1,
    interior = 2,
    halo = 3 
    end = 4
```